src/main/java/io/gatling/mojo/RecoverEnterprisePluginException.java

```diff
@@ -0,0 +1,94 @@
+
+/*
+ * Copyright 2011-2022 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gatling.mojo;
+
+import io.gatling.plugin.EmptyChoicesException;
+import io.gatling.plugin.exceptions.*;
+import io.gatling.plugin.model.Simulation;
+import java.util.stream.Collectors;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.Log;
+
+public class RecoverEnterprisePluginException {
+
+  @FunctionalInterface
+  public static interface EnterprisePluginExceptionFunction<R> {
+    R apply() throws EnterprisePluginException;
+  }
+
+  static <R> R handle(EnterprisePluginExceptionFunction<R> f, Log log) throws MojoFailureException {
+    try {
+      return f.apply();
+    } catch (UnsupportedJavaVersionException e) {
+      final String msg =
+          e.getMessage()
+              + "\nIn order to target the supported Java bytecode version, please use the following maven setting:\n"
```

Comment:
```suggestion
              + "\nIn order to target the supported Java bytecode version, please use the following Maven setting:\n"
```

Reply:
We may also want to add a link to the Maven documentation for this option ( https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html)?